libs/src/services/identity/index.js

```diff
@@ -248,8 +248,11 @@ class IdentityService {
   }
 
   async relay (contractRegistryKey, contractAddress, senderAddress, encodedABI, gasLimit) {
+    // Only probabilistically capture 1 out 2 relay captchas
+    const shouldCaptcha = Boolean(Math.round(Math.random()))
```

Comment:
nit, slight preference for this to be

const RELAY_CAPTCHA_SAMPLE_RATE = 0.5

// below
const shouldCaptcha = Math.random() > RELAY_CAPTCHA_SAMPLE_RATE

Reply:
Reasonable. Will update

Reply:
Will use < actually so when we update the rate the probability is still intact